### PR TITLE
lasso examples

### PR DESCRIPTION
**Pull request by krajsek**
## Description

This PR removes PyTorch and Numpy examples from example/lasso/demo.py

Issue/s resolved: #528

## Changes proposed:
Removed PyTorch and Numpy Lasso examples

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## Due Diligence

- [ ] All split configurations tested
- [ ] Multiple dtypes tested in relevant functions
- [ ] Documentation updated (if needed)
- [ ] Updated changelog.md under the title "Pending Additions"

#### Does this change modify the behaviour of other functions? If so, which?
no

### CODE REVIEW

File: heat/regression/lasso.py [hunk 1]
```diff
@@ -116,17 +116,29 @@ def fit(self, X, y):
         # Get number of model parameters
         _, n = X.shape
 
+        assert len(y.shape) <= 2, "y.shape needs to be 1 or 2 dimensional"
+        assert len(X.shape) == 2, "X.shape needs to be 2 dimensional"
```
Comment (coquelin77) [inline heat/regression/lasso.py:120]: should these be raises instead?

Comment (Markus-Goetz) [reply heat/regression/lasso.py:120]: Would also claim that should be consistent exceptions

Comment (krajsek) [reply heat/regression/lasso.py:120]: Is there a predefined error type for shapes in Python?

Comment (Markus-Goetz) [reply heat/regression/lasso.py:120]: Should be ValueError

**Pull request merged**